staging/src/k8s.io/apiserver/pkg/storage/value/encrypt/envelope/grpc_service.go

```diff
@@ -166,3 +161,22 @@ func (g *gRPCService) Encrypt(plain []byte) ([]byte, error) {
 	}
 	return response.Cipher, nil
 }
+
+func (g *gRPCService) interceptor(
+	ctx context.Context,
+	method string,
+	req interface{},
+	reply interface{},
+	cc *grpc.ClientConn,
+	invoker grpc.UnaryInvoker,
+	opts ...grpc.CallOption,
+) error {
+	if !strings.HasSuffix(method, ".KeyManagementService/Version") {
```

Comment:
I was thinking more of letting this be a check in the api package (e.g. `IsVersionCheckMethod(method)`) to prevent other packages from needing to know about the service/method name

Reply:
@liggitt, I like the idea of IsVersionCheckMethod, though I am not sure what you mean by the "api package". Could you, please, clarify this for me.

Reply:
The `k8s.io/apiserver/pkg/storage/value/encrypt/envelope/v1beta1` package is the one that knows the service/method name corresponding to the version check. It would make sense to contain that knowledge to that package.

Reply:
@liggitt  Makes sense. PTAL.